### PR TITLE
Chopin and Ragtime model training and output with loss of 0.0049

### PR DESCRIPTION
**Pull request by gjmulder**
@Skuldur thanks for a great musical project!

I pulled down some MIDI files from [Kunstderfuge](http://kunstderfuge.com) and switched your code to use CUDA and Batch Normalisation to get a loss of 0.0049 after 644 epochs on my NVidia 1080Ti.

The resultant weights are too large for Github, so I uploaded them to [my Dropbox].(https://www.dropbox.com/s/scmblb97o77ceb8/weights_chopin_ragtime.hdf5.bz2?dl=0):

### CODE REVIEW

**Comment by Skuldur** [comment]
Hi,

Might I ask why this is a pull request?
